sw-emulator/lib/cpu/src/cpu.rs

```diff
@@ -734,15 +915,19 @@ impl<TBus: Bus> Cpu<TBus> {
         self.global_int_en = false;
 
         self.write_pc(next_pc);
-        println!(
-            "handle_trap: cause={:x}, mtval={:x}, pc={:x}, next_pc={:x}",
-            cause, info, pc, next_pc
-        );
+        // println!(
+        //     "handle_trap: cause={:x}, mtval={:x}, pc={:x}, next_pc={:x}",
+        //     cause, info, pc, next_pc
+        // );
         Ok(())
     }
 
     //// Handle external interrupts
     fn handle_external_int(&mut self, irq: u8) -> StepAction {
+        #[cfg(test)]
+        {
```

Comment:
I think we need to fix the bug from https://github.com/chipsalliance/caliptra-mcu-sw/commit/e624b327fedd18fa8c3a41f752ba0a1df4d2aaef#diff-1218360c85f22cf0efa8a33c84bd38598bfe7736003aa354d6c93fccc6f5bb00R791 -- ensuring we read the vector table with the right privilege level

Reply:
Fixed, I believe.